server/utils/assessmentUtils.ts

```diff
@@ -114,6 +114,16 @@ const informationSetAsNotReceived = (assessment: Assessment): boolean => {
   return false
 }
 
+const getSections = (assessment: Assessment) => {
+  let { sections } = Assess
+
+  if (informationSetAsNotReceived(assessment)) {
+    sections = sections.filter(section => section.name !== 'AssessApplication')
```

Comment:
Would changing `'AssessApplication'` to `AssessApplication.name` work/be safer?

Reply:
Great idea! I'll try that 👍